internal/stanza/register.go

```diff
@@ -15,9 +15,12 @@
 package stanza
 
 import (
-	// Register parsers for stanza-based log receivers
+	// Register parsers and transformers for stanza-based log receivers
 	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/parser/json"
 	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/parser/regex"
 	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/parser/severity"
 	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/parser/time"
+	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/transformer/metadata"
+	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/transformer/restructure"
+	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/transformer/router"
```

Comment:
I was unable to make linter happy with comment above `transformer` section. At least one of two linters has been always raising some issues:

```
Check License finished successfully
internal/stanza/register.go: Import groups are not in the proper order: ["Third party" "Third party"]
```
or
```
register.go:22: File is not `goimports`-ed with -local github.com/open-telemetry/opentelemetry-collector-contrib (goimports)
        _ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/parser/time"
```
or
```
register.go:23: File is not `gofmt`-ed with `-s` (gofmt)

```

I tried three different approaches
```
	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/parser/time"
	// comment here
	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/transformer/metadata"
```

```
	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/parser/time"

	// comment here
	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/transformer/metadata"
```

```
	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/parser/time"
        
	// comment here (preceding line is tab indented)
	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/transformer/metadata"
```

Reply:
No need to split into two groups - looks good to me like this.